cmd/oras/root/attach.go

```diff
@@ -77,6 +77,12 @@ Example - Attach an artifact with manifest annotations:
 Example - Attach file 'hi.txt' and add manifest annotations:
   oras attach --artifact-type doc/example --annotation "key=val" localhost:5000/hello:v1 hi.txt
 
+Example - Attach file 'hi.txt' and format output with JSON:
+  oras attach --artifact-type doc/example localhost:5000/hello:v1 hi.txt --format json
+
+Example - Attach file 'hi.txt' and format output with Go template:
+  oras attach --artifact-type doc/example localhost:5000/hello:v1 hi.txt --format go-template --template {{.digest}}
```

Comment:
It is better to quote the template.
```suggestion
  oras attach --artifact-type doc/example localhost:5000/hello:v1 hi.txt --format go-template --template "{{.digest}}"
```

Reply:
Same comment for other commands